### PR TITLE
Added participants/kick/mute/unmute to cb_conferences

### PR DESCRIPTION
**Pull request by danielfinke**
Moved actions into POST to conference/id

Changed conference_participants to list comp

#1249 against master

### CODE REVIEW

**Comment by fenollp** [comment]
Check fails because you introduced trailing whitespace. Please remove all trailing whitespace :)

File: applications/crossbar/src/modules/cb_conferences.erl [hunk 1]
```diff
@@ -216,6 +228,94 @@ load_conference(DocId, Context) ->
...
+    AuthDoc = cb_context:auth_doc(Context),
+    ConfDoc = cb_context:doc(load_conference(Id, Context)),
+    ConfOwnerId = wh_json:get_value(<<"owner_id">>, ConfDoc),
+    AuthOwnerId = wh_json:get_value(<<"owner_id">>, AuthDoc),
+    % Abort if the user is not room owner
+    Resp = case ConfOwnerId of
+        AuthOwnerId ->
+            Conference = whapps_conference:set_id(Id, whapps_conference:new()),
+            perform_conference_action(Conference, Action, ParticipantId);
+        _ -> <<"user_not_owner">>
```
Comment (k-anderson) [inline applications/crossbar/src/modules/cb_conferences.erl]: @danielfinke 

This should not be a successful response ;)

Could you update it to use:
https://github.com/2600hz/kazoo/blob/master/applications/crossbar/src/cb_context.erl#L983
and only successful from the above case clause?

File: applications/crossbar/src/modules/cb_conferences.erl [hunk 2]
```diff
@@ -216,6 +228,94 @@ load_conference(DocId, Context) ->
...
+    end,
+    crossbar_doc:handle_json_success(wh_json:set_value(<<"resp">>, Resp, wh_json:new()), Context).
+
+perform_conference_action(Conference, <<"mute">>, ParticipantId) ->
```
Comment (k-anderson) [inline applications/crossbar/src/modules/cb_conferences.erl]: @danielfinke 
This function should have a spec

File: applications/crossbar/src/modules/cb_conferences.erl [hunk 3]
```diff
@@ -216,6 +228,94 @@ load_conference(DocId, Context) ->
...
+           ,{<<"Fields">>, []}
+           ,{<<"Conference-ID">>, ConfId}
+           | wh_api:default_headers(?APP_NAME, ?APP_VERSION)
+          ],
+    ReqResp = whapps_util:amqp_pool_collect(Req
+                                            ,fun wapi_conference:publish_search_req/1
+                                            ,{'ecallmgr', 'true'}
+                                           ),
+    case ReqResp of
+        {'error', _} -> cb_context:add_system_error('conference_search_error', Context);
```
Comment (k-anderson) [inline applications/crossbar/src/modules/cb_conferences.erl]: @danielfinke 
How would you feel about a more typical reply such as 
https://github.com/2600hz/kazoo/blob/master/applications/crossbar/src/cb_context.erl#L1018
or just the not_found?

**Pull request merged**